sn_node/src/put_validation.rs

```diff
@@ -178,12 +175,6 @@ impl Node {
         &mut self,
         signed_spends: Vec<SignedSpend>,
     ) -> Result<CmdOk, ProtocolError> {
-        // Prevents someone from crafting large Vec and slowing down nodes.
-        if signed_spends.len() > MAX_SIGNED_SPENDS_LENGTH {
-            warn!("Discarding incoming DbcSpend PUT as it contains more than {MAX_SIGNED_SPENDS_LENGTH} SignedSpends");
-            return Err(ProtocolError::MaxNumberOfSpendsExceeded);
-        }
-
```

Comment:
This was superfluous as we only take two SignedSpends maximum in the end in `signed_spend_validation`. 
Also this check was useless because an attacker could have bypassed this by adding thousands of spends, two by two...